Framework/Crystal/src/PredictSatellitePeaks.cpp

```diff
@@ -319,9 +329,17 @@ void PredictSatellitePeaks::predictOffsets(int indexModulatedVector, V3D offsets
       continue;
 
     IPeak_uptr iPeak = Peaks->createPeak(Qs, 1);
```

Comment:
I think the code would be significantly clearer if you picked better names than `peak` and `iPeak` for the two objects. Without fully understanding the relation between the two I can't offer good names, but these are too similar to each other to not be confusing.